colearn_examples/grpc/mnist_grpc.py

```diff
@@ -87,7 +91,7 @@ def prepare_data_loaders(location: str,
 @FactoryRegistry.register_model_architecture(model_tag, [dataloader_tag])
 def prepare_learner(data_loaders: Tuple[PrefetchDataset, PrefetchDataset, PrefetchDataset],
                     steps_per_epoch: int = 100,
-                    vote_batches: int = 10,
+                    vote_batches: int = 1,
```

Comment:
The vote_score and test_score are directly effected by that. Until now the model has been tested batch wise (this means that in this case the set was split into 10 sets) on the test and vote set. The score e.g. accuracy was then calculated over the mean of those 10 sets. the reason of having a test set (same is valid for the vote set) in this case is to evaluate the model on the whole data, and not split it again into multiple parts. If we set this variable to 1 - the metric calculation is done properly.

My thought here would be to remove the variable from the `register_model_architecture` and set it in the `keras_learner` itself to avoid that it will be overwritten in the future?